gradle-plugin/build.gradle

```diff
@@ -21,7 +21,6 @@ dependencies {
   implementation "org.jetbrains.kotlin:kotlin-gradle-plugin-api:$KOTLIN_VERSION"
   implementation "org.jetbrains.kotlin:kotlin-gradle-plugin:$KOTLIN_VERSION"
   implementation "io.github.classgraph:classgraph:$CLASS_GRAPH_VERSION"
-  implementation "xerces:xercesImpl:$XERCES_VERSION"
```

Comment:
Just a question for what did we use this @rachelcarmena?

Reply:
To parse the XML file of the private IDEA plugins repository: https://github.com/arrow-kt/arrow-meta/blob/main/gradle-plugin/src/main/kotlin/arrow/meta/plugin/gradle/InstallIdeaPlugin.kt#L50